src/cascadia/TerminalSettingsModel/WslDistroGenerator.cpp

```diff
@@ -23,6 +24,18 @@ static constexpr wchar_t RegKeyDistroName[] = L"DistributionName";
 using namespace ::Microsoft::Terminal::Settings::Model;
 using namespace winrt::Microsoft::Terminal::Settings::Model;
 
+static bool isWslDashDashCdAvailableForLinuxPaths() noexcept
+{
+    OSVERSIONINFOEXW osver{};
+    osver.dwOSVersionInfoSize = sizeof(osver);
+    osver.dwBuildNumber = 19041;
+
+    DWORDLONG dwlConditionMask = 0;
+    VER_SET_CONDITION(dwlConditionMask, VER_BUILDNUMBER, VER_GREATER_EQUAL);
+
+    return VerifyVersionInfoW(&osver, VER_BUILDNUMBER, dwlConditionMask) != FALSE;

```

Comment:
i know it seems weird to check this when Terminal's min version is 19041 already, but... 1.12 still runs on 18362 and this is  backport candidate. I don't want to blow up folks on that version combo